neo4j/api.py

```diff
@@ -51,6 +51,10 @@
 TRUST_SYSTEM_CA_SIGNED_CERTIFICATES = "TRUST_SYSTEM_CA_SIGNED_CERTIFICATES"  # Default
 TRUST_ALL_CERTIFICATES = "TRUST_ALL_CERTIFICATES"
 
+DATABASE_NAME_SYSTEM = "system"
+DATABASE_NAME_NEO4J = "neo4j"
+DATABASE_NAME_USE_DEFAULT = None
```

Comment:
USE_DEFAULT sounds like a flag but is the name of the default database

Reply:
That is correct it should be

`DEFAULT_DATABASE = None`
`SYSTEM_DATABASE = "system"`